identity-service/src/notifications/utils.js

```diff
@@ -333,13 +333,32 @@ const shouldSendEmail = (
   )
 }
 
+const getSupporters = async (receiverUserId) => {
+  console.log('do not call')
```

Comment:
Log

Reply:
ahh yes, srry was trying to get the sinon stub to work - will remove